README.md

```diff
@@ -2,23 +2,14 @@
 A JavaScript library to visualize process execution data. Diagrams are displayed from BPMN files. With additional display options for execution data (highlight some transitions, counters, and more). With interactive capacities (mouse hover, click).
 
 [![Build](https://github.com/bonitasoft-labs/bpmn-visu-js/workflows/Build/badge.svg)](https://github.com/bonitasoft-labs/bpmn-visu-js/actions)
-
-[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md) 
-[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)  
+ 
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](CONTRIBUTING.md)
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
 
 
 Supported Browsers: Chrome, Firefox, Safari, Edge
 
-# Build and run
-
-## Requirements
-
-> node >= 12.x
-
-
-## Starting the project
-`npm install`
-
-`npm run watch`
+# Requirements
 
-You can now access the project on http://localhost:10001
+- > node >= 12.x
+- typescript
```

Comment:
not a requirements for run, just for dev, I suggest to move this to the contributing guide

Reply:
Moved